lib/multi.c

```diff
@@ -1300,6 +1300,54 @@ static CURLcode multi_do_more(struct connectdata *conn, int *complete)
   return result;
 }
 
+static bool multi_handle_timeout(struct Curl_easy *data,
+                                 struct curltime *now,
+                                 bool *stream_error,
+                                 CURLcode *result,
+                                 bool connect_timeout)
+{
+  time_t timeout_ms;
+  timeout_ms = Curl_timeleft(data, now, connect_timeout);
+
+  if(timeout_ms < 0) {
+
```

Comment:
(Really pedantic)  This whitespace looks a bit odd - particularly, as the comment belongs with the if rather than the following block.

Reply:
I think this is standard for CURL. I've moved the comment block to the top of the function which should help.

Reply:
Yes, that was what I was referring to - we'd added a blank line between the if and the comment..